### ISSUE TITLE
A correction to the description of ":pa[ckadd][!] {name}" in doc/repeat.txt

### ISSUE THREAD

**Issue by lkintact** [vim/vim#2234]
The description says: "All the files matching the pattern `pack/*/opt/{name}/plugin/**/*.vim`  will be sourced", but it seems the correct pattern is  `pack/*/opt/{name}/**/*.vim`.

**Comment by brammool** [comment]
What makes you think that?

**Comment by lkintact** [comment]
I tried installing the `vim-javascript` package (https://github.com/pangloss/vim-javascript), for which I
1. Added the line `packadd vim-javascript-master`  to the _vimrc file 
2. Copied the contents of `vim-javascript-master.zip` (the folder `vim-javascript-master`, which contains 2 files and 6 subfolders) to `C:\Program Files\Vim\vimfiles\pack\others\opt`.

According to the currently indicated pattern only the files in `C:\Program Files\Vim\vimfiles\pack\others\opt\vim-javascript-master\plugin` and its subfolders should be accessible for sourcing, but in fact everything in `C:\Program Files\Vim\vimfiles\pack\others\opt\vim-javascript-master` and its subfolders is.
